modules/apps/object/object-test/src/testFunctional/tests/macros/VisualizeObjectDisplayPageTemplate.macro

```diff
@@ -0,0 +1,37 @@
+definition {
+
+	macro createObjectPageTemplate {
+		VisualizeObjectDisplayPageTemplate.openPageTemplate();
+
+		Click(locator1 = "VisualizeObjectDisplayPageTemplate#SELECT_BLANK_OBJECT_DISPLAY_PAGE_TEMPLATE");
+
+		Type(
+			locator1 = "VisualizeObjectDisplayPageTemplate#NAME_OBJECT_DISPLAY_PAGE_TEMPLATE",
+			value1 = "Object Display Page Template");
```

Comment:
This value should be passed as a parameter and not inserted directly into the macro.